lua/lspconfig/cssls.lua

```diff
@@ -21,7 +21,77 @@ configs[server_name] = {
   };
   docs = {
     description = [[
-https://github.com/vscode-langservers/vscode-css-languageserver-bin
+
+    
+https://www.npmjs.com/package/vscode-langservers-extracted
+
+`css-languageserver` can be installed via `npm`:
+
+```sh
+npm i -g vscode-langservers-extracted
+```
+    
+Neovim does not currently include built-in snippets. vscode-css-language-server only provides completions when snippet support is enabled. To enable completion, install a snippet plugin and add the following override to your language client capabilities during setup.
+
+```lua
+--Enable (broadcasting) snippet capability for completion
+local capabilities = vim.lsp.protocol.make_client_capabilities()
+capabilities.textDocument.completion.completionItem.snippetSupport = true
+
+require'lspconfig'.cssls.setup {
+  capabilities = capabilities,
+}
```

Comment:
Pretty much everything after this point is unecessary IMO

Reply:
As a new user, I added the example as I found the original section  confusing/insufficient (with the nested defaults). I’m a fan of a cut and paste example but obviously go with what makes sense from your POV.

Reply:
I think the cut/paste example is overly specific to your usecase.  Also one of the blocks basically duplicates what docgen does

Reply:
Hi, commit https://github.com/neovim/nvim-lspconfig/pull/971/commits/61ba064065b301657cc3ad31e26fd89795044d41 should have it as you desire.

Reply:
Lines 45 to 66 still need to be removed